content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -81,6 +81,7 @@ front-proxy-ca          Dec 28, 2029 23:36 UTC   9y              no
 The command shows expiration/residual time for the client certificates in the
 `/etc/kubernetes/pki` folder and for the client certificate embedded in the kubeconfig files used
 by kubeadm (`admin.conf`, `controller-manager.conf` and `scheduler.conf`).
+If missing, `super-admin.conf` will not cause an error.
```

Comment:
```suggestion
Missing of `super-admin.conf` will not cause an error.
```

Reply:
the original is valid. "missing of" is also valid since "missing" is a valid noun...yet not so commonly used, from my observations. another way to write this would be "The absence of `super-admin.conf` will not cause an error".

Reply:
this is not a binding comments. :)